support-frontend/assets/components/socialShare/socialShare.jsx

```diff
@@ -98,5 +101,10 @@ export default function SocialShare(props: PropTypes) {
       </p>
     </button>
   );
-
 }
+
+SocialShare.defaultProps = {
+  referralCode: '',
```

Comment:
since the field is optional, should the default be `undefined`?

Reply:
or can we just not have defaultProps here?

Reply:
✅